packages/react-common/src/moratorium-banner.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+export const CovidMoratoriumBanner: React.FC<{ locale?: string }> = ({
+  locale,
+}) => {
+  return locale === "es" ? (
+    <>
+      <b>Actualización COVID-19: </b>
```

Comment:
Weird, I was going to say we should use `<strong>` instead of `<b>` but after reading the [MDN docs on `<b>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b) it actually seems like `<b>` is fine here--give it a read, I'm curious what you think.

Reply:
Oh interesting! Hmm honestly the docs aren't super clear— like what really is the difference between "boldness" and "importance"? But yeah it does sound like <b> makes the most sense here.

Reply:
Ok sounds good! Also BTW, HTML is actually a subset of Markdown, so you need to put backticks around your `<b>` or else it will just render bold text, as it did in your comment above ;)

Reply:
oh haha weird! lol